client/scss/layouts/_login.scss

```diff
@@ -1,18 +1,28 @@
 // overrides default nice padding defined in variables.scss
 $desktop-nice-padding: 100px;
 
-.login,
-.login .content-wrapper {
-  background-color: $color-grey-1;
-  color: $color-grey-3;
-}
-
 .login {
-  margin-inline-start: 0;
-  height: 100%;
+  background: theme('colors.primary.DEFAULT');
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0px 390px 170px;
```

Comment:
Try to use the theme spacing values here. Makes it feel more consistent and the spacing values are in REMs which is better for users that may have zoomed in.